# CMD_WPT
# The vehicle will fly straight to the desired waypoint. By default hold position if no further waypoints received. Note, the F3-OPS code will monitor progress towards waypoints and issue new ones.

time    timestamp
uint8   vehicleID
uint8   commandID
float64 northing
float64 easting
float64 altitude
---
bool    success